applications/smart-distancing/config-jetson.ini

```diff
@@ -13,7 +13,7 @@ ImageSize: 300,300,3
 ModelPath: 
 ClassID: 0
 MinScore: 0.25
-DistThreshold: 0.1
+DistThreshold: 150
```

Comment:
What's the metric? 150cm?
It would be nicer to left a short comment about the metric.
e.g. ;cm